dist/quill.js

```diff
@@ -6767,30 +6767,22 @@ Line = (function(superClass) {
     }
     _.each(formats, (function(_this) {
       return function(value, name) {
-        var exclude, excludeFormats, format;
+        var exclude, format;
         format = _this.doc.formats[name];
-        if (format == null) {
+        if (!((format != null) && format.isType(Format.types.LINE))) {
```

Comment:
Yeah it is crap.  I can't even follow the logic and I wrote the coffeescript.  This line of coffeescript compiled to that code.
```
     return unless format? && format.isType(Format.types.LINE)
```  
I changed it to this
```
return if !format? || !format.isType(Format.types.LINE)
```
now compiles to this.
```
if ((format == null) || !format.isType(Format.types.LINE)) {
```